sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,366 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Managing Azure Machine Learning Workspace with Managed Network\n",
+    "\n",
+    "**Requirements** - In order to benefit from this tutorial, you will need:\n",
+    "- A basic understanding of Machine Learning\n",
```

Comment:
This is not really needed. Do you mean basic understanding of Azure Machine Learning concepts? (workspaces, etc.)

Reply:
removed